### PR TITLE
Improve generated config initializer files

### PR DESCRIPTION
**Pull request by jonatack**
- Fix grammar.
- Improve readability/understanding.
- Wrap lines at 80 characters.

[skip ci]

### CODE REVIEW

File: railties/lib/rails/generators/rails/app/templates/config/initializers/active_record_belongs_to_required_by_default.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Be sure to restart your server when you modify this file.
 
-# Require `belongs_to` associations by default. This is a new Rails 5.0 default,
-# so introduced as a config to ensure apps made with earlier versions of Rails aren't affected when upgrading.
+# Require `belongs_to` associations by default. This is a new Rails 5.0
+# default, so it is introduced as a configuration file to ensure that apps made
```
Comment (thomasfedb) [inline railties/lib/rails/generators/rails/app/templates/config/initializers/active_record_belongs_to_required_by_default.rb]: Would change `configuration file` to `configuration option` and also consider changing `configuration` to `config` as that is the term used inside Rails.

File: railties/lib/rails/generators/rails/app/templates/config/initializers/callback_terminator.rb [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # Be sure to restart your server when you modify this file.
 
-# Do not halt callback chains when a callback returns false. This is a new Rails 5.0 default,
-# so introduced as a config to ensure apps made with earlier versions of Rails aren't affected when upgrading.
+# Do not halt callback chains when a callback returns false. This is a new
+# Rails 5.0 default, so it is introduced as a configuration file to ensure that
```
Comment (thomasfedb) [inline railties/lib/rails/generators/rails/app/templates/config/initializers/callback_terminator.rb]: Same here.

File: railties/lib/rails/generators/rails/app/templates/config/initializers/active_record_belongs_to_required_by_default.rb (hunk 1 above)
Comment (jonatack) [inline railties/lib/rails/generators/rails/app/templates/config/initializers/active_record_belongs_to_required_by_default.rb]: Sure, thanks for the feedback. Updated and squashed.

**Comment by jonatack** [comment]
Updated and squashed.

**Pull request merged**